doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -462,6 +462,62 @@ Displaying information about notations
 
    :flag:`Printing All` to disable other elements in addition to notations.
 
+.. cmd:: Print Notation @string
+
+   Displays information about the previously reserved notation string
+   :token:`string`, namely the level at which it was reserved and its
+   associativity.
+
+   .. example::
+
+      .. coqtop:: all
+
+         Reserved Notation "x $ y" (at level 123, right associativity).
+         Print Notation "_ $ _".
+
+   Variables can be indicated with either "_" or names, as long as these can not
+   be confused with notation symbols. When confusion may arise, for example with
+   notation symbols that are entirely made up of letters, use single quotes to
+   delimit those symbols.
+
+   .. example::
+
+      .. coqtop:: all
+
+         Reserved Notation "x 'mod' y" (at level 40, no associativity).
+         Print Notation "_ mod _".
+         Print Notation "x 'mod' y".
+
+   There must always be (at least) a space between notation symbols and
+   arguments, even when the notation format does not include those spaces.
+
+   .. example::
+
+      .. coqtop:: all
+
+         Reserved Notation "/ x /" (at level 0, format "/ x /").
+         Fail Print Notation "/x/".
+         Print Notation "/ x /".
+
+   Recursive notations can also be printed, as the following example
+   illustrates.
+
+   .. example::
+
+      .. coqtop:: all
+
+        Reserved Notation "( x , y , .. , z )" (at level 0).
+        Print Notation "( _ , _ , .. , _ )".
+
+
+   .. exn:: :token:`string` cannot be interpreted as a known notation. Make sure
+            that symbols are surrounded by spaces and that holes are explicitly
+            denoted with "_".
```

Comment:
Is there some other place in the refman that should list this new exception?

Reply:
If it's already in the manual once, I wouldn't add it again.  A lot of messages apply to multiple commands or tactics.  The way the doc handles these today is not too great--it will be a project to get all the messages into the doc (requiring on code changes) and I think the presentation should be different for messages that apply to more than one command/tactic.  Nor will the error messages index work well with repeated messages.

Reply:
@jfehrle I think the question is: Should we add this to an error index?

Reply:
This is fixed now! For future reference, the problem was that the documentation of the error message didn't have a body, only a title.